### ISSUE TITLE
decouple adaptation-interval from chrouter-update-interval

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#8567]
sensible travel time aggregation requires a low sampling interval (maybe 1-30s) whereas CH preprocessing should be done at 300s or higher.

adding a new option **--device.rerouting.ch-interval** should solve this.

**Comment by namdre** [comment]
@behrisch: this might have been a problem for tapas all along and might lead to significant speedup

**Comment by behrisch** [comment]
We are doing A* with tapas

**Comment by namdre** [comment]
Yes. We stopped using CH because it didn't give the expected speed-up
